crates/op-rbuilder/src/flashtestations/builder_tx.rs

```diff
@@ -0,0 +1,581 @@
+        nonce: u64,
+    ) -> Result<Recovered<OpTransactionSigned>, secp256k1::Error> {
+        let calldata = IBlockBuilderPolicy::verifyBlockBuilderProofCall {
+            version: self.builder_proof_version,
+            blockContentHash: block_content_hash,
+        }
+        .abi_encode();
+        // Create the EIP-1559 transaction
+        let tx = OpTypedTransaction::Eip1559(TxEip1559 {
+            chain_id: ctx.chain_id(),
+            nonce,
+            gas_limit,
+            max_fee_per_gas: ctx.base_fee().into(),
+            max_priority_fee_per_gas: 0,
+            to: TxKind::Call(self.builder_policy_address),
+            input: calldata.into(),
+            ..Default::default()
+        });
+        self.tee_service_signer.sign_tx(tx)
+    }
+
+    /// Computes the block content hash according to the formula:
+    /// keccak256(abi.encode(parentHash, blockNumber, timestamp, transactionHashes))
+    /// https://github.com/flashbots/rollup-boost/blob/main/specs/flashtestations.md#block-building-process
+    fn compute_block_content_hash(
+        transactions: Vec<OpTransactionSigned>,
+        parent_hash: B256,
+        block_number: u64,
+        timestamp: u64,
+    ) -> B256 {
+        // Create ordered list of transaction hashes
+        let transaction_hashes: Vec<B256> = transactions
+            .iter()
+            .map(|tx| {
+                // RLP encode the transaction and hash it
+                let mut encoded = Vec::new();
+                tx.encode_2718(&mut encoded);
+                keccak256(&encoded)
+            })
+            .collect();
+
+        // Create struct and ABI encode
+        let block_data = BlockData {
+            parentHash: parent_hash,
+            blockNumber: U256::from(block_number),
+            timestamp: U256::from(timestamp),
+            transactionHashes: transaction_hashes,
+        };
+
+        let encoded = block_data.abi_encode();
+        keccak256(&encoded)
+    }
+
+    fn simulate_register_tee_service_tx<ExtraCtx: Debug + Default>(
+        &self,
+        ctx: &OpPayloadBuilderCtx<ExtraCtx>,
+        evm: &mut OpEvm<
+            &mut State<StateProviderDatabase<impl StateProvider>>,
+            NoOpInspector,
+            PrecompilesMap,
+        >,
+    ) -> Result<TxSimulateResult, BuilderTransactionError> {
+        let nonce = get_nonce(evm.db_mut(), self.tee_service_signer.address)?;
+
+        let register_tx = self.signed_register_tee_service_tx(
+            self.attestation.clone(),
+            ctx.block_gas_limit(),
+            ctx.base_fee(),
+            ctx.chain_id(),
+            nonce,
+        )?;
+        let ResultAndState { result, state } = match evm.transact(&register_tx) {
+            Ok(res) => res,
+            Err(err) => {
+                if err.is_invalid_tx_err() {
+                    warn!(target: "flashtestations", %err, "register tee service tx failed");
+                    return Ok(TxSimulateResult::default());
```

Comment:
why is it okay to ignore the error here?

Reply:
they are not fatal errors should be handled separately (nonce, balance, gas issues)